src/test/java/org/apache/commons/jxpath/ri/model/BeanModelTestCase.java

```diff
@@ -44,14 +44,23 @@
 public abstract class BeanModelTestCase extends JXPathTestCase {
     private JXPathContext context;
 
-    public void setUp() {
+    @Override
+    public void setUp() throws Exception {
 //        if (context == null) {
-            context = JXPathContext.newContext(createContextBean());
-            context.setLocale(Locale.US);
-            context.setFactory(getAbstractFactory());
+        super.setUp();
+        System.setProperty("jxpath.class.allow", "*");
+        context = JXPathContext.newContext(createContextBean());
+        context.setLocale(Locale.US);
+        context.setFactory(getAbstractFactory());
 //        }
     }
 
+    @Override
+    public void tearDown() throws Exception {
+        super.tearDown();
```

Comment:
Very minor, should call super.tearDown at the end of the method not the beginning.

Reply:
changed